docs/installation/server-config/security.rst

```diff
@@ -0,0 +1,163 @@
+.. _ksql-security:
+
+Securing KSQL
+-------------
+
+To connect to a secured Kafka cluster, Kafka client applications must provide their security credentials. In the same way,
+KSQL is configured so that the KSQL servers are authenticated and authorized, and data communication is encrypted when
+communicating with the Kafka cluster. KSQL can be configured for:
+
+- :ref:`SSL for encryption <kafka_ssl_encryption>`
+- :ref:`SASL for authentication <kafka_sasl_auth>`
+- :ref:`HTTPS for Confluent Schema Registry <schema_registry_http_https>`
+
+-------------------------------------------------
+Using KSQL with an Apache Kafka Secured with ACLs
+-------------------------------------------------
+
+You can use KSQL with Apache Kafka clusters that are secured with ACLs. The behavior depends on whether the cluster is
+interactive or non-interactive.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Interactive KSQL clusters
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Interactive KSQL clusters require that the KSQL user has open access to create, read, write and delete topics and
+use any consumer group.
+
+The required ACLs are:
+
+- *DESCRIBE_CONFIGS* permission on the *CLUSTER*.
```

Comment:
It must be clear whether cluster refers to Kafka cluster vs. KSQL cluster.

Reply:
`Kafka *CLUSTER*`?

Reply:
@big-andy-coates : What is "CLUSTER" referring to here?  Is that a specific security role? Or ...?

Reply:
`CLUSTER` is referring to the exact enum value in the code that would be used to create the ACL, same as  `DESCRIBE_CONFIGS`.  We should just switch to  straight `describe_configs` and `cluster`, but I think/hope the above will make more sense to someone actually implementing this.

`Kafka *CLUSTER*` works.

Reply:
@big-andy-coates instead of an asterisk, should we use command syntax (if this referring to actual values in the code)? e.g. `\`\CLUSTER\`\`

Reply:
Yep. Don't see why not.